src/vs/platform/update/common/update.config.contribution.ts

```diff
@@ -9,6 +9,7 @@ import { ConfigurationScope, Extensions as ConfigurationExtensions, IConfigurati
 import { Registry } from '../../registry/common/platform.js';
 
 const configurationRegistry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration);
+// --- START POSITRON ---
```

Comment:
Nit, here and elsewhere: should use the standard casing for comment fences (`Start Positron`)

Reply:
I keep forgetting this!